packages/verified-fetch/src/utils/request-headers.ts

```diff
@@ -0,0 +1,86 @@
+import { type CatOptions } from '@helia/unixfs'
+import { CodeError } from '@libp2p/interface'
+import { type ExporterOptions } from 'ipfs-unixfs-exporter'
+
+export function getHeader (headers: HeadersInit | undefined, header: string): string | undefined {
```

Comment:
This is only used in the `getRequestRange` function, no need to export it.

If the input to `getRequestRange` is [simplified to Headers] this function can be removed entirely?

Reply:
I think we will continue to have a need for a getHeader function throughout this application though?

Reply:
resolving this as I think it makes sense for a normalized header getting function to exist in a request-headers utility file